akka-typed-tests/src/test/scala/docs/akka/typed/IntroSpec.scala

```diff
@@ -128,13 +128,8 @@ class IntroSpec extends TypedSpec {
         Immutable[akka.NotUsed] {
           (_, _) ⇒ Unhandled
         } onSignal {
```

Comment:
For me it's more about if the previous brace is `}` or `)`, so `).onSignal {` but ` } onSignal { `.
Scala style guide does indeed recommend to not do infix for named methods... We're in DSL land though so it would be OK if we want it IMHO.

I'd like 

```
onMessage { } 

onMessage { 
  // ...
} onSignal { 
  // ...
}
```

with . it becomes confusing if it should be:

```
onMessage { 
}
.onSignal { 
}

// or 

onMessage { 
  // ... 
}.onSignal { 
  // ... 
}
```